### DISCUSSION TITLE
Playbackspeed-Controls

### DISCUSSION THREAD

**Discussion by flowhorn** [alexmercerind/dart_vlc#168] [Ideas]
I think it would be really useful and nice if it was possible to set a playback speed. E.g. 1.25/1.5/2.0x

**Comment by alexmercerind** [comment]
@flowhorn, this feature is already available.

https://github.com/alexmercerind/dart_vlc/blob/d13d06e5e4f1536e6a533c5657793390d78f10d0/ffi/lib/src/player.dart#L246-L252

```dart
player.setRate(1.25);
```

**Reply by flowhorn** [reply]
Thank you for your quick reply!
